README.md

```diff
@@ -1,6 +1,6 @@
 # npm on IPFS
 
-![Alt text](./img/npm-on-ipfs.svg)
+![npm on IPFS logo](./img/npm-on-ipfs.svg)
```

Comment:
nicpick, can you make it centered. Just like libp2p https://github.com/libp2p/js-libp2p?